### PR TITLE
Preprocessing resume project

### PR DESCRIPTION
**Pull request by murbar**

### CODE REVIEW

**Comment by murbar** [comment]
A couple notes
- I just used my bartending resume so I didn't have to worry about coming up with content and I could concentrate on the code
- I went mobile first, with breakpoints where they made sense for larger devices

File: less/index.less [hunk 1]
```diff
@@ -54,6 +54,214 @@ table {
...
 
+
+
+// my styles
+
+// MVP - incorporate variables for colors and font stacks
+@body-font: 'Lato', sans-serif;
+@heading-font: 'PT Serif', serif;
+@body-text-color: #5A371E;
+
```
Comment (sabrinagear) [inline less/index.less]: good use of variables

File: less/index.less [hunk 2]
```diff
@@ -54,6 +54,214 @@ table {
...
+	letter-spacing: 0.2rem;
+}
+.input-style() {
+	width: 100%;
+	margin: 1rem 0;
+	padding: 1rem;
+	font-size: 1em;
+	border: 1px solid @body-text-color;
+	border-radius: 0.5rem;
+}
```
Comment (sabrinagear) [inline less/index.less]: great use of responsive units.

File: less/index.less [hunk 3]
```diff
@@ -54,6 +54,214 @@ table {
...
+			font-size: 1.5rem;
+			display: block;
+			text-transform: capitalize;
+		}
+	}
+	.qualifications li {
+		font-size: 90%;
+	}
+	.gig {
+		display: flex;
```
Comment (sabrinagear) [inline less/index.less]: great job with nesting

File: less/index.less [hunk 4]
```diff
@@ -54,6 +54,214 @@ table {
...
+		flex-flow: wrap;
+		margin-bottom: 3rem;
+		justify-content: space-between;
+		.name {
+			font-weight: bold;
+			order: 1;
```
Comment (sabrinagear) [inline less/index.less]: good job with flex

File: less/index.less [hunk 5]
```diff
@@ -54,6 +54,214 @@ table {
...
+			left: calc(53% - 660px);
+		}
+	}
+}
+@media (min-width: 1100px) {
+	.wrapper {
+
+		.spoon {
+			left: calc(50% - 660px);
+		}
```
Comment (sabrinagear) [inline less/index.less]: Joel, excellent job. Simple layout that is clean and completely responsive. Followed a logical layout. Great use of your preprocessor. I have no constructive feedback. Looks great.

Comment (murbar) [reply less/index.less]: Thank you!

**Pull request merged**